pkgs/development/tools/continuous-integration/laminar/default.nix

```diff
@@ -50,31 +43,23 @@ in stdenv.mkDerivation rec {
     cp  ${js.vue-router}  js/vue-router.min.js
     cp  ${js.ansi_up}     js/ansi_up.js
     cp  ${js.Chart}       js/Chart.min.js
-    cp  ${css.bootstrap}  css/bootstrap.min.css
   '';
+
   postInstall = ''
-    mv $out/usr/share $out
-    mkdir $out/bin
-    mv $out/usr/{bin,sbin}/* $out/bin
-    rmdir $out/usr/{bin,sbin}
-    rmdir $out/usr
+    mv $out/usr/share/* $out/share/
+    rmdir $out/usr/share $out/usr
 
     mkdir -p $out/share/doc/laminar
     pandoc -s ../UserManual.md -o $out/share/doc/laminar/UserManual.html
-  '' + lib.optionalString (customConfig != null) ''
-    cp ${customConfig} /etc/etc/laminar.conf
-  '' + (if enableSystemd then ''
-    sed -i "s,/etc/,$out/etc/," $out/lib/systemd/system/laminar.service
-    sed -i "s,/usr/sbin/,$out/bin/," $out/lib/systemd/system/laminar.service
-  '' else ''
-    rm -r $out/lib # it contains only systemd unit file
-  '');
+    rm -rf $out/lib # remove upstream systemd units
+    rm -rf $out/etc # remove upstream config file
```

Comment:
I am find with removing the files if they provide no value. Dow e have proper manpages for the configuration or is the configuration file a good example?

Reply:
That configuration file contains decent documentation for the variables, but it’s completely redundant to the man page `man laminard`.